test/swirl.jl

```diff
@@ -0,0 +1,16 @@
+@testset "Swirl Operation" begin
+    img = [0 0 0 0 0
+           0 0 1 0 0
+           0 1 0 0 0
+           0 0 1 0 0
+           0 0 0 0 0]
+    expected = [ NaN NaN       0.0        0.0      NaN
+                 0.0 0.431797  0.476449   0.431797 NaN
+                 0.0 0.476449  0.0        0.393894 0.0
+                 NaN 0.431797  0.0825553  0.0      0.0
+                 NaN 0.0       0.0        NaN      NaN]
+    res = swirl(img, 1, 10, 1)
+    replace!(res, NaN=>0.0)
+    replace!(expected, NaN=>0.0)
+    @test isapprox(res, expected; atol=0.1)
```

Comment:
There's `nearlysame` https://github.com/JuliaImages/ImageTransformations.jl/blob/4d89e5cecbf115d6b25976f12ac7a32c92a5bfc0/test/runtests.jl#L10-L11 that can be used here.

Reply:
issue arose mainly from here
```
julia> expected[2,2]
0.431797

julia> res[2,2]
0.4317972472093078
```

Reply:
```
nearlysame(x, y, atol = 0.0) = isapprox(x, y; atol = atol) || (isnan(x) & isnan(y))
nearlysame(A::AbstractArray, B::AbstractArray; atol = 0.0) = all(map((a,b,atol)-> nearlysame(a, b, atol), A, B, atol))
```
I think this would be better